app/assets/javascripts/reports.js

```diff
@@ -1,26 +0,0 @@
-$(function() {
```

Comment:
please remove the reference to the reports.js, .e.g. at https://github.com/theforeman/foreman/blob/d7f8cdd651cf1060bf28abd93dbceac8e72eccd1/app/views/config_reports/show.html.erb#L1

(maybe we can remove the charts js as well if we migrated away to react charts?)